skbio/stats/ordination/_canonical_correspondence_analysis.py

```diff
@@ -9,22 +9,25 @@
 from __future__ import absolute_import, division, print_function
 
 import numpy as np
+import pandas as pd
+from scipy.linalg import svd, lstsq
 
-from ._base import Ordination, OrdinationResults
+from skbio._base import OrdinationResults
 from ._utils import corr, svd_rank, scale
 from skbio.util._decorator import experimental
 
 
-class CCA(Ordination):
-    r"""Compute constrained (also known as canonical) correspondence
+@experimental(as_of="0.4.0")
+def cca(y, x, scaling=1):
+    r"""Compute canonical (also known as constrained) correspondence
     analysis.
 
     Canonical (or constrained) correspondence analysis is a
     multivariate ordination technique. It appeared in community
     ecology [1]_ and relates community composition to the variation in
     the environment (or in other factors). It works from data on
-    abundances or counts of individuals and environmental variables,
-    and outputs ordination axes that maximize niche separation among
+    abundances or counts of samples and constrains variables,
+    and outputs ordination axes that maximize samples separation among
```

Comment:
Glad that we dropped "niche".  May want to change this to "sample separation though"